profiling-commands.go

```diff
@@ -47,77 +45,6 @@ const (
 	ProfilerRuntime    ProfilerType = "runtime"    // Include runtime metrics
 )
 
-// StartProfilingResult holds the result of starting
-// profiler result in a given node.
-type StartProfilingResult struct {
-	NodeName string `json:"nodeName"`
-	Success  bool   `json:"success"`
-	Error    string `json:"error"`
-}
-
-// StartProfiling makes an admin call to remotely start profiling on a
-// standalone server or the whole cluster in case of a distributed setup.
-//
-// Deprecated: use Profile API instead
```

Comment:
This will be extremely annoying for warp, since it will be unable to terminate benchmarks when using "autoterm" or otherwise interrrupted. It really need separate start/stop commands.

`--serverprof=cpu,mem` is extremely convenient.

Reply:
okay then it is not a deprecated API, will keep it back.